src/bin/signal_handler.rs

```diff
@@ -14,28 +14,28 @@
 
 #[cfg(unix)]
 mod imp {
-    use std::time::Duration;
     use std::sync::Arc;
-    use std::sync::atomic::AtomicBool;
 
     use rocksdb::DB;
+    use libc;
 
     use tikv::server::Msg;
     use tikv::util::transport::SendCh;
     use prometheus::{self, Encoder, TextEncoder};
 
+    // Real-time signals
+    const TOGGLE_PROF_SIG: libc::c_int = 40;
```

Comment:
Is there any plan to switch to RPC like HTTP/GRPC later? It's handy to use signal for notify, but it's not the best way to add more and more signals for all that kind of things.